src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig

```diff
@@ -710,7 +710,7 @@
                           <i class="material-icons">help</i>
                           <p class="alert-text" data-title="{{ trans('Read more', {}, 'AdminProducts')|raw }}">
                             {{ trans('Tags are meant to help your customers find your products via the search bar.', {}, 'AdminProducts')|raw }}
-                            <strong>{{ trans('Read more', {}, 'AdminProducts')|raw }}</strong>
+                            <a title="{{ trans('Read more', {}, 'AdminProducts')|raw }}">{{ trans('Read more', {}, 'AdminProducts')|raw }}</a>
```

Comment:
It is more user friendly to have hand on a link instead of bold text and default cursor :)